bot/exts/filters/antispam.py

```diff
@@ -103,6 +103,7 @@ async def upload_messages(self, actor_id: int, modlog: ModLog) -> None:
             mod_alert_message += content
 
         await modlog.send_log_message(
+            content=", ".join(str(m.id) for m in self.members),  # quality-of-life improvement for mobile moderators
```

Comment:
Make this a set, so the same user_id isn't sent more than once.

```suggestion
            content=", ".join({str(m.id) for m in self.members}),  # quality-of-life improvement for mobile moderators
```

Reply:
Actually, members is already a frozenset, so this can be ignored.